controllers/rhmi/prometheusRules.go

```diff
@@ -19,6 +19,14 @@ var (
 	}
 )
 
+func getAddonName(installation *integreatlyv1alpha1.RHMI) string {
+	if installation.Spec.Type == "managed-api" {
+		return "managed-api-service"
+	} else {
+		return "integreatly-operator"
+	}
```

Comment:
A small suggestion just in case there is a different entry to the Type, we can handle that with an empty string (or something else if you think there is a better value) rather than always defaulting to `integreatly-operator` 

```suggestion
      switch installation.Spec.Type {
	case "managed-api":
		return "managed-api-service"
	case "integreatly-operator":
		return "integreatly-operator"
	default:
		return ""
	}
```

Reply:
Thank you @laurafitzgerald. Adding a default case makes more sense.

I'll keep that as a note for future. Also I've committed the suggestion patch.